### DISCUSSION TITLE
Rollback

### DISCUSSION THREAD

**Discussion by Me5uT** [ant-design/ant-design#46971] [General]
![Ekran görüntüsü 2024-01-14 235003](https://github.com/ant-design/ant-design/assets/82406976/c3f4c72d-f219-421c-87ee-c93c6fbe7c5f)

After this update, type errors started in almost the entire application!!! Am I the only one experiencing these problems?

**Comment by afc163** [comment]
This commit has not been released yet, could you provide a reproduce?

**Comment by crazyair** [comment]
> This commit has not been released yet, could you provide a reproduce?

https://codesandbox.io/p/sandbox/ji-ben-yong-fa-antd-5-12-8-forked-l8g7mw?file=%2Fdemo.tsx%3A4%2C59

https://stackblitz.com/edit/react-ptbnwy?file=demo.tsx
